lib/litmus_paper/app.rb

```diff
@@ -100,6 +105,14 @@ class App < Sinatra::Base
       _text 500, "Server Error"
     end
 
+    def _cache
+      @cache ||= LitmusPaper::Cache.new(
+        "/run/shm",
```

Comment:
I don't know that we can ensure this path will always exist on all hosts. `/run/shm` exists in Debian Wheezy, but it's `/dev/shm` in Debian Jessie and above.

Can we make this a configurable path?

Reply:
```
ls -l /run/shm
lrwxrwxrwx 1 root root 8 May 23 07:15 /run/shm -> /dev/shm
```
looks like a symlink on jessie and higher

Reply:
Mmm. I still think this should probably be configurable. Though this is probably a fine default.